src/lib/y2firstboot/dialogs/wsl_product_selection.rb

```diff
@@ -81,24 +91,40 @@ def dialog_content
             ),
             VSpacing(2),
             # TRANSLATORS:
-            Label(_("The WSL GUI pattern provides some needed packages for\n" \
-              "a better experience with graphical applications in WSL.")),
+            Left(Label(_("The WSL GUI pattern provides some needed packages for\n" \
+              "a better experience with graphical applications in WSL."))),
             VSpacing(1),
             # TRANSLATORS: check box label
             Left(CheckBox(Id(:wsl_gui_pattern),
               _("Install WSL GUI pattern (requires registration)"),
-              wsl_gui_pattern))
+              wsl_gui_pattern)),
+            VSpacing(2),
+            # TRANSLATORS:
+            Left(Label(_("The WSL systemd pattern provides wsl.conf adjustment\n" \
+              "and init symlink for systemd enablement in WSL."))),
+            VSpacing(1),
+            # TRANSLATORS: check box label
+            Left(CheckBox(Id(:wsl_systemd_pattern),
+              _("Install WSL systemd pattern (requires registration)"),
+              wsl_systemd_pattern))
           )
         )
       end
+      # rubocop:enable Metrics/AbcSize
 
       def help_text
-        # TRANSLATORS: help text (1/2)
+        # TRANSLATORS: help text (1/3)
         _("<p>Select the product to use with Windows Subsystem for Linux (WSL). " \
           "Some products might require registration.</p>") +
-          # TRANSLATORS: help text (2/2)
-          _("<p>To use graphical programs in WSL you need to install the WSL GUI pattern. " \
-              "In that case the system needs to be registered as well.</p>")
+          # TRANSLATORS: help text (2/3)
+          _("<p>For smoother experience with graphical programs in WSL " \
+              "the WSL GUI pattern provides recommended config, tools and libraries. " \
+              "In that case the system needs to be registered as well.</p>") +
+          # TRANSLATORS: help text (3/3)
+          _("<p>For enablement of systemd in WSL the WSL systemd pattern provides wsl.conf " \
+              "and /sbin/init adjustments. " \
+              "In that case the system needs to be registered as well. " \
+              "Also be aware that systemd enablement is in effect only after relaunch.</p>")
```

Comment:
I would reword the last sentence a bit: "Reboot is required for enabling systemd".

Reply:
problem is that in wsl it is not reboot. That is why I use relaunch...so I would use your sentence with Relaunch